grails-app/services/com/causecode/mongo/UpdateEmbeddedInstancesService.groovy

```diff
@@ -52,8 +53,17 @@ class UpdateEmbeddedInstancesService {
             String currentDomainClassName = domainClass.name
             Map domainInfoMap = [(currentDomainClassName): []]
 
+            List<String> embeddedFields
+            try {
+                embeddedFields = domainClass.clazz.embedded
```

Comment:
Please add a comment for this change. Mention that the Class has been deprecated and causing this issue.

Reply:
Added